### ISSUE TITLE
Contrast too low

### ISSUE THREAD

**Issue by juyrjola** [City-of-Helsinki/servicemap#287]
Tekstin ja taustavärin välinen kontrasti ei täytä WCAG-standardin vaatimuksia kohdissa, joissa on vaaleansininen teksti joko valkoisella tai harmaalla pohjalla. Myöskään palveluiden selausvalikossa tekstin ja taustan kontrastiero ei ole riittävä valikon aktiivisena olevassa kohdassa, musta teksti harmaalla pohjalla.

WCAG: 1.4.3 Kontrasti (minimi): Tekstin visuaalisella esityksellä ja tekstiä esittävissä kuvissa kontrastisuhde on vähintään 4,5:1, paitsi seuraavissa tapauksissa (ks. tarkemmin
http://www.w3.org/Translations/WCAG20-fi/).

**Comment by kallelongjuhani** [comment]
It seems the accent blue is greatly lacking in contrast, so we need to radically change it. Let's make the following changes:

### Global changes to colour values

Main blue
From #33a8ff
to #005cff

Main blue highlighted
From #50c7ff
to #004ED8

Light blue background
From #e2ebfd
to #eef0ff

### Specific changes

Grey text on light blue background has poor contrast. Change the text color in the following places:

#navigation-region.overlay ul li.info-box
#personalisation .personalisation-container h2
#service-cart.overlay ul li.info-box

Color from #797776
to #333333


### Updated images

A couple of image assets will need replacing because of the colour change.

Logos: https://drive.google.com/folderview?id=0Bz_jZaEUKzZ9cVZHVm5xZlFrLXc&usp=sharing

Tour vignette: https://drive.google.com/folderview?id=0Bz_jZaEUKzZ9VDN2QWFHUkhmZWs&usp=sharing
